src/tx-format/transaction.md

```diff
@@ -51,27 +49,27 @@ enum ReceiptType : uint8 {
     TransferOut = 8,
     ScriptResult = 9,
     MessageOut = 10,
-    Mint = 11
-    Burn = 12
+    Mint = 11,
+    Burn = 12,
 }
 ```
 
-| name               | type                        | description                                          |
-|--------------------|-----------------------------|------------------------------------------------------|
-| `gasPrice`         | `uint64`                    | Gas price for transaction.                           |
-| `gasLimit`         | `uint64`                    | Gas limit for transaction (including predicate gas). |
-| `maturity`         | `uint32`                    | Block until which tx cannot be included.             |
-| `scriptLength`     | `uint16`                    | Script length, in instructions.                      |
-| `scriptDataLength` | `uint16`                    | Length of script input data, in bytes.               |
-| `inputsCount`      | `uint8`                     | Number of inputs.                                    |
-| `outputsCount`     | `uint8`                     | Number of outputs.                                   |
-| `witnessesCount`   | `uint8`                     | Number of witnesses.                                 |
-| `receiptsRoot`     | `byte[32]`                  | Merkle root of receipts.                             |
-| `script`           | `byte[]`                    | Script to execute.                                   |
-| `scriptData`       | `byte[]`                    | Script input data (parameters).                      |
-| `inputs`           | [Input](./input.md)`[]`     | List of inputs.                                      |
-| `outputs`          | [Output](./output.md)`[]`   | List of outputs.                                     |
-| `witnesses`        | [Witness](./witness.md)`[]` | List of witnesses.                                   |
+| name               | type                        | description                            |
+|--------------------|-----------------------------|----------------------------------------|
+| `gasPrice`         | `uint64`                    | Gas price for transaction.             |
```

Comment:
Do we also want to move `gas_price` to policies as part of the preparation for the multi-dimension prices?

With the current approach, we will include it in each transaction, but later, with storage price and on-chain gas price, we can remove it or treat it like a tip.

Reply:
true, moving it here would make it easier to deprecate later on with an automated pricing model.

Reply:
The field is still here=D